task.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# создаём каталог task с вложенными директориями
+# task
+#   dir1
+#   dir2
+#   dir3
+#       dir4
+
+mkdir -p task/dir1 task/dir2 task/dir3/dir4
+
+# изменяем текущую директорию на task
+
+cd ./task
+
+# создаём пустой файл task/dir2/empty
+
+touch dir2/empty
+
+# создаём файл task/dir2/hello.sh с таким содержанием:
+# #!/bin/bash
+# echo "$1, привет!"
+
+echo '#!/bin/bash
+echo "$1, привет!"' > dir2/hello.sh
+
+# устанавливаем для task/dir2/hello.sh права rwxrw-r--
+
+sudo chmod 764 dir2/hello.sh 
```

Comment:
Если ты владелец файла, то для изменения прав судо не нужен
```suggestion
chmod 764 dir2/hello.sh 
```